src/routes/movement-detection/MovementDetection.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useCallback } from 'react';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import map from 'lodash/map';
+import filter from 'lodash/filter';
+import reduce from 'lodash/reduce';
+import property from 'lodash/property';
+import Aside from './Aside/Aside';
+import Info from './Info/Info';
+import Sensor from './Sensor/Sensor';
+import addImgUrl from '../../actions/movementDetections/img';
+import {
+  updateSensors,
+  disarmSensors,
+} from '../../actions/movementDetections/sensors';
+import s from './MovementDetection.css';
+
+function MovementDetection() {
+  const dispatch = useDispatch();
+  const { imgUrl, sensors, isDisarmed } = useSelector(
```

Comment:
As we figured out, it's better to use `useSelector` thrice in cases like this one.